### PR TITLE
[backup] move away from gPRC storage server

### PR DESCRIPTION
**Pull request by msmouse**
implemented the other API needed on the HTTP server.

## Motivation

We are about ready to remove gRPC server entirely.

### Have you read the [Contributing Guidelines on pull requests](https://github.com/libra/libra/blob/master/CONTRIBUTING.md#pull-requests)?

yes

## Test Plan

tests

### CODE REVIEW

**Comment by lightmark** [comment]
@bors-libra r+
kudos to @msmouse

File: config/src/config/test_data/random.complete.node.config.toml [hunk 1]
```diff
@@ -61,7 +61,7 @@ max_chunk_limit = 1000
 max_timeout_ms = 120000
 
 [storage]
-address = "127.0.0.1:6184"
```
Comment (msmouse) [inline config/src/config/test_data/random.complete.node.config.toml:64]: @davidiw  is it okay to fix it like this? context: we removed "address" for grpc, and renamed "simple_address" for the SimpleStorageService to "address", which originally had the value of 6666

Comment (davidiw) [reply config/src/config/test_data/random.complete.node.config.toml:64]: that's how we roll!

yes, it is okay

Comment (lightmark) [reply config/src/config/test_data/random.complete.node.config.toml:64]: 6666 is just a magic number I thought of. I think it's the same for 6184...

**Pull request closed without merging**